allred.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Probably don't want this here

Reply:
yea, ill axe it when i get closer to landing. it hasn't lost its utility yet